src/statistics_test.py

```diff
@@ -143,10 +143,11 @@ def adjustScoresToData(self, dataToAdjustTo):
         originalScores = getAllScores(self.markedTests)
         bestScores = originalScores
         bestChiSquared = binned_data(bestScores).chiSquaredTest(dataToAdjustTo)
+        addingConstant = binned_data.binRangeSize/5
```

Comment:
Nice! I hadn't considered about how hill climb would be impacted by ranges of marks different to our test dataset. +-2 might be fine for ours on ~100 marks, but on a 10 mark quiz it would obviously be no good. 2% seems a good amount to increase/decrease by.